models/issue_reaction.go

```diff
@@ -40,9 +41,28 @@ func (opts *FindReactionsOptions) toConds() builder.Cond {
 	if opts.CommentID > 0 {
 		cond = cond.And(builder.Eq{"reaction.comment_id": opts.CommentID})
 	}
+	if opts.CommentID == -1 {
```

Comment:
How about use `else if` or a `switch` here?

Reply:
the only thin wich would make sence is else if

Reply:
done